src/Android/Xamarin.Android/Samples/Network Analysis/ClosestFacility/ClosestFacility.cs

```diff
@@ -0,0 +1,207 @@
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.Tasks.NetworkAnalysis;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ArcGISRuntime.Samples.ClosestFacility
+{
+    [Activity(Label = "ClosestFacility")]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Closest facility (interactive)",
+        "Network Analysis",
+        "Demonstrates how to solve a Closest Facility Task to find the closest route between a facility (hospital) and a incident (black cross). Tap to find the route to the nearest hospital.",
+        "")]
+    public partial class ClosestFacility : Activity
+    {
+        // Map view that will be used
+        private MapView _myMapView = new MapView();
+
+        // Holds locations of hospitals around San Diego
+        private List<Facility> _facilities;
+
+        // Graphics overlays for facilities and incidents
+        private GraphicsOverlay _facilityGraphicsOverlay;
+
+        // Symbol for facilities
+        private PictureMarkerSymbol _facilitySymbol;
+
+        private GraphicsOverlay _incidentGraphicsOverlay;
+
+        // Black cross where user clicked
+        private MapPoint _incidentPoint;
+
+        // Symbol for the incident
+        private SimpleMarkerSymbol _incidentSymbol;
+
+        // Used to display route between incident and facility to mapview
+        private SimpleLineSymbol _routeSymbol;
+
+        // Solves task to find closest route between an incident and a facility
+        private ClosestFacilityTask _task;
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Closest Facility";
+
+            // Initialize the app
+            Initialize();
+
+            // Create the UI
+            CreateLayout();
+        }
+
+        private void CreateLayout()
+        {
+            // Create a new layout for the entire page
+            var layout = new LinearLayout(this) { Orientation = Orientation.Vertical };
+
+            // Add the map view to the layout
+            layout.AddView(_myMapView);
+
+            // Show the layout in the app
+            SetContentView(layout);
+        }
+
+        private void Initialize()
+        {
+            // Hook up the DrawStatusChanged event.
+            _myMapView.DrawStatusChanged += OnDrawStatusChanged;
+
+            // Load the basemap.
+            Map map = new Map(Basemap.CreateLightGrayCanvasVector());
+            _myMapView.Map = map;
+
+            // Create a ClosestFacilityTask using the San Diego Uri.
+            _task = ClosestFacilityTask.CreateAsync(new Uri("http://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ClosestFacility")).Result;
+
+            // Spatial reference to be used
+            SpatialReference _spatialReference = SpatialReferences.WebMercator;
```

Comment:
I don't know that this adds a lot of value; consider replacing instances of this variable with `SpatialReferences.WebMercator` directly.

Reply:
If you choose not to remove the variable, then remove the underscore prefix. We don't use a prefix for local variables.